CMakeLists.txt

```diff
@@ -132,7 +132,7 @@ SET(CMAKE_MODULE_PATH
 IF(DEFINED GEPETTO_GUI_HAS_PYTHONQT AND GEPETTO_GUI_HAS_PYTHONQT)
   FIND_PACKAGE(PythonQt REQUIRED COMPONENTS QtAll REQUIRED)
 ELSE()
-  FIND_PACKAGE(PythonQt COMPONENTS QtAll REQUIRED)
+  FIND_PACKAGE(PythonQt COMPONENTS QtAll)
```

Comment:
But then, we can have `PythonQt_FOUND` be true while `PythonQt_QtAll_FOUND` is false.

So below, shouldn't we use `if (PythonQt_FOUND ANDPythonQt_QtAll_FOUND)`